update README and remove beta status (#91)
triton-inference-server/openvino_backend: README.md

# OpenVINO Backend

> **Note: OpenVINO backend is now considered of general availability. It is co-maintained by Nvidia and Intel.**

Reviewer: ```suggestion
**Note: The OpenVINO backend has been promoted out of Beta and is now ready for production use. It is co-maintained by Nvidia and Intel.**
```

Reviewer 2: I don't think we need this note at all. Simple just state in the description below that this repo is co-maintained by Intel and NVIDIA and remove all mentions of beta or GA. We can make an official announcement in the release notes.

Reviewer: will make the change part of release notes / announcments.

> **Note: OpenVINO backend is now considered of general availability. It is co-maintained by Nvidia and Intel.**

Reviewer: ```suggestion
**Note: The OpenVINO backend is co-maintained by Nvidia and Intel.**
```
